src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -26,6 +26,24 @@ public class AcspController {
     @Autowired
     private TransactionService transactionService;
 
+    @PostMapping("/transactions/{" + TRANSACTION_ID_KEY + "}/acsp")
+    public ResponseEntity<Object> createAcspData(
+            @PathVariable(TRANSACTION_ID_KEY) String transactionId,
+            @RequestHeader(value = ERIC_ACCESS_TOKEN) String requestId,
+            @RequestHeader(value = ERIC_IDENTITY) String userId,
+            @RequestBody AcspDataDto acspData) {
+        LOGGER.info("received POST request to save acsp data");
+        try {
+            var transaction = transactionService.getTransaction(requestId, transactionId);
+            return acspService.createAcspRegData(transaction, acspData, requestId, userId);
+        } catch (ServiceException e) {
+            LOGGER.info("Error saving data " + e.getMessage());
```

Comment:
change info to error.  Make sure this "Error saving data " logger is distinguishable from PUT end point.